content/references/rippled-api/public-rippled-methods/server-info-methods/server_info.md

```diff
@@ -115,6 +115,7 @@ An example of a successful response:
       "pubkey_node": "n94UE1ukbq6pfZY9j54sv2A1UrEeHZXLbns3xK5CzU9NbNREytaa",
       "pubkey_validator": "n9KM73uq5BM3Fc6cxG3k5TruvbLc8Ffq17JZBmWC4uP4csL4rFST",
       "server_state": "proposing",
+      "server_state_duration_us": 166972201508,
```

Comment:
The "full" state (required to be "proposing" for validators) is the current state, there's been >1800 transitions to it, so we can't have been in the current full state for the entire duration since the other 1800+ durations would also have taken up time. So make this "92762334"